jetty-infinispan/infinispan-common/src/main/java/org/eclipse/jetty/session/infinispan/InfinispanSessionData.java

```diff
@@ -19,6 +20,7 @@
  * pool and thus these threads have no knowledge of the correct classloader to
  * use.
  */
+@SerializeWith(InfinispanSessionDataSerializer.class)
```

Comment:
You're sure that this annotation will have no effect in the remote case?

Reply:
I am not sure, but I run the remote tests and it doesn't seem to have any effect on those